lib/pxconfig.js

```diff
@@ -245,16 +247,11 @@ function pxInternalConfig() {
             'webp',
             '.jar',
         ],
-
-        MONITOR_MODE: {
-            MONITOR: 0,
-            BLOCK: 1,
-        },
     };
 }
 
 /* to be defined by the initiating user or dynamically via configuration service */
-function pxDefaultConfig(PX_INTERNAL) {
```

Comment:
Can't believe that this was the way it was written before lol, why PX_INTERNAL was a parameter when it is constant...

Reply:
I KNOW. 😡